lib/std/os.zig

```diff
@@ -310,7 +310,7 @@ pub fn read(fd: fd_t, buf: []u8) ReadError!usize {
             EINTR => continue,
             EINVAL => unreachable,
             EFAULT => unreachable,
-            EAGAIN => if (std.event.Loop.instance) |loop| {
+            EAGAIN => if (std.io.mode == .evented) if (std.io.mode == .evented) if (std.event.Loop.instance) |loop| {
```

Comment:
you added this twice?

Reply:
I did it once, then did a find + replace in the document. Whoops